server/container/metrics/metrics-dropwizard/src/main/java/org/apache/james/metrics/dropwizard/ESReporterConfiguration.java

```diff
@@ -62,8 +62,7 @@ public ESReporterConfiguration(Optional<String> host, Optional<Integer> port, bo
     }
 
     public String getHostWithPort() {
-        Preconditions.checkState(host.isPresent());
-        Preconditions.checkState(port.isPresent());
+        Preconditions.checkState(host.isPresent() && port.isPresent());
```

Comment:
Sorry to answer on this a little late, but that's not our coding style. We prefer to split preconditions, so we can know exactly which one is failing.